### ISSUE TITLE
Unknown field 'presentationHeight' on type 'CloudinaryAssetFluid'.

### ISSUE THREAD

**Issue by dhruuuuuv** [cloudinary-devs/gatsby-transformer-cloudinary#40]
Hi,

I'm getting the following error when trying to use this plugin to query fluid images.

I'm using Gatsby version: 2.15.20, and the latest version of the plugin from the repo.

Thanks!

**Comment by dhruuuuuv** [comment]
If I open the file with the error in (cloudinary-asset-fragments.js) in the cache, I can see the following code snippet - maybe that will shed some light on what's happening.


```
export const cloudinaryAssetFluidLimitPresentationSize = graphql`
  fragment CloudinaryAssetFluidLimitPresentationSize on CloudinaryAssetFluid {
    maxHeight: presentationHeight
    maxWidth: presentationWidth
  }
`
```

**Comment by Chuloo** [comment]
Hi thanks for this @dhruuuuuv a fix for this is in #36 kindly check if it's working on your end now 👍

**Comment by Chuloo** [comment]
Closing this out now. Please re-open this issue if you're still facing this issue.
